src/main/java/pl/javastart/task/Reader.java

```diff
@@ -0,0 +1,35 @@
+package pl.javastart.task;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Reader {
+
+    public static List<Product> readProduct(String fileName) throws FileNotFoundException {
+        Scanner scanner = new Scanner(new File(fileName));
```

Comment:
Pamiętaj o zamykaniu pliku, albo najlepiej wprowadź try-with-resources https://youtu.be/LRwSHA9YDgY